Adding a new guide about caching in browser tests (#10774)
DataDog/documentation: content/en/synthetics/guide/browser-test-cache.md

---
title: Avoiding cache issues in browser tests
kind: guide
further_reading:
    - link: 'synthetics/browser_tests'
      tag: 'Documentation'
      text: 'Configure a Browser Test'
    - link: '/synthetics/browser_tests/actions'
      tag: 'Documentation'
      text: 'Create Browser Test Steps'
    - link: 'https://www.datadoghq.com/blog/test-creation-best-practices/'
      tag: 'Blog'
      text: 'Best practices for creating end-to-end tests'

---

> Browsers are killed after each test execution meaning there should not be any cache related issues on the client side.

Author: ```suggestion
Browsers are killed after every test execution. This ensures your tests do not face cache related issues on the client side.
```

> Browsers are killed after every test execution. This ensures your tests do not face cache related issues on the client side.

Author: ```suggestion
## Browser tests

Browsers are killed after every test execution. This ensures your browser tests do not face cache related issues on the client side.

## API tests

You can leverage unique [local variables][1] to perform cache busting on your API tests.
```

## Further Reading

> {{< partial name="whats-next/whats-next.html" >}}

Author: ```suggestion
{{< partial name="whats-next/whats-next.html" >}}

[1]: /synthetics/api_tests/http_tests?tab=requestoptions#create-local-variables
```

---
title: Avoiding cache issues in browser tests
kind: guide
further_reading:
    - link: 'synthetics/browser_tests'
      tag: 'Documentation'
      text: 'Configure a Browser Test'
    - link: '/synthetics/browser_tests/actions'
      tag: 'Documentation'
      text: 'Create Browser Test Steps'
    - link: 'https://www.datadoghq.com/blog/test-creation-best-practices/'
      tag: 'Blog'
      text: 'Best practices for creating end-to-end tests'

---

## Browser tests

Browsers are killed after every test execution. This ensures your browser tests do not face cache related issues on the client side.

## API tests

> You can leverage unique [local variables][1] to perform cache busting on your API tests.

Author: ```suggestion
You can leverage [local variables][1] to generate a random string to send with your payload and perform cache busting on your API tests.
```
